### DISCUSSION TITLE
IBAX Sandık Sorunsalı

### DISCUSSION THREAD

**Discussion by hes91** [IBAX-io/go-ibax#734] [Soru Cevap - Türkçe]
Sandıktaki puanlar stabil değil.Bir gün başka bir gün başka puan veriyor.Neye göre kimine fazla verirken kimine daha az veriyor?

**Comment by Yusufknk** [comment]
Sandıktan çıkan puanlar da tamamen şans faktörü devrededir. Herhangi bir kişiye/şahsa odaklı değildir.
